plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -198,8 +199,9 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_LARGE_FB:
 			return "Allocate a large framebuffer to be compliant with GS memory (Prevents FMV flickering).\n"
 				"It increases GPU/memory requirements.\n\n"
-				"Note: It should be disabled for Armored Core, Destroy All Humans, and Gran Turismo.\n"
-				"It will amplify RAM/VRAM spikes. Other games might be affected as well.";
+				"Note: It should be disabled for Armored Core, Destroy All Humans, Gran Turismo and many others .\n"
+				"It will amplify RAM/VRAM spikes with a chance of not improving anything for you.\n\n"
+				"RECOMMENDED to leave this disabled in most cases as it only helps select titles.";
```

Comment:
Likewise.

Reply:
I think this one needs revision. I would expect that to a normal end-user will read this option is in 'most cases fine especially on a good rig'.